LiteDB/Engine/EngineState.cs

```diff
@@ -51,5 +51,12 @@ public bool Handle(Exception ex)
 
             return true;
         }
+
+        public BsonValue ReadTransform(string collection, BsonValue value)
+        {
+            if (_settings?.ReadTransform == null) return value;
```

Comment:
Using pattern-matching to check nullability is safer and better.

```suggestion
            if (_settings?.ReadTransform is null) return value;
```